sample-plugin/src/test/java/io/jenkins/tools/bom/sample/DeclarativePipelineTest.java

```diff
@@ -0,0 +1,60 @@
+package io.jenkins.tools.bom.sample;
+
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import org.junit.ClassRule;
+import org.junit.Rule;
+import org.junit.Test;
+import org.jvnet.hudson.test.BuildWatcher;
+import org.jvnet.hudson.test.JenkinsRule;
+
+public class DeclarativePipelineTest {
+    @ClassRule
+    public static BuildWatcher buildWatcher = new BuildWatcher();
+
+    @Rule
+    public JenkinsRule r = new JenkinsRule();
+
+    @Test
+    public void smokes() throws Exception {
+        final WorkflowRun run = runPipeline(m(
+                "pipeline {",
+                "  agent none",
+                "  stages {",
+                "    stage('Example') {",
+                "      steps {",
+                "        echo 'Hello world'",
```

Comment:
May as well

```suggestion
                "        example(x: 'some value')",
```

and for that matter just move this to `ExampleStepTest`.